http4s-server/src/main/scala/com/avast/server/toolkit/http4s/Http4sServerConfig.scala

```diff
@@ -0,0 +1,37 @@
+package com.avast.server.toolkit.http4s
+
+import java.util.concurrent.TimeUnit
+
+import com.avast.server.toolkit.http4s.Http4sServerConfig.SocketOptions
+import org.http4s.blaze.channel
+import org.http4s.server.defaults
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+final case class Http4sServerConfig(
+  listenAddress: String = "127.0.0.1",
+  listenPort: Int = 8080,
```

Comment:
I would prefer `80` here - it's something that one could expect.

Reply:
i would prefer not having this default here at all ... just make people to make a decision about something as important as this

Reply:
I think guys are right that this value should not have a default so in the next PR update there will be none.